azure-pipelines/publish.yml

```diff
@@ -0,0 +1,47 @@
+name: $(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+  branches:
+    include:
+      - main
+pr: none
+
+resources:
+  repositories:
+    - repository: templates
+      type: github
+      name: microsoft/vscode-engineering
+      ref: main
+      endpoint: Monaco
+
+parameters:
+  - name: publishPackage
+    displayName: 🚀 Publish windows-process-tree
```

Comment:
Just to clarify, this won't publish to the `@vscode` org, correct?

Reply:
Correct, based on the information from the `package.json` file.

Reply:
Is this package windows specific? If so, should it be packages on windows?